cg/services/analysis_starter/configurator/file_creators/nextflow/params_file/raredisease.py

```diff
@@ -75,40 +78,34 @@ def _get_data_analysis_type(self, case_id: str) -> str:
         sample: Sample = self.store.get_samples_by_case_id(case_id=case_id)[0]
         return sample.application_version.application.analysis_type
 
-    def _get_target_bed(self, case_id: str, analysis_type: str) -> str:
-        """
-        Return the target bed file from LIMS or use default capture kit for WHOLE_GENOME_SEQUENCING.
-        Raises:
-            ValueError if not capture kit can be assigned to the case.
-        """
-        target_bed_file: str = self._get_target_bed_from_lims(case_id=case_id)
-        if not target_bed_file:
-            if analysis_type == AnalysisType.WGS:
-                return DEFAULT_CAPTURE_KIT
-            raise ValueError("No capture kit was found in LIMS")
-        return target_bed_file
-
-    def _get_target_bed_from_lims(self, case_id: str) -> str | None:
+    def _get_target_bed_from_lims(self, case_id: str) -> str:
         """
-        Get target bed filename from LIMS.
+        Get target bed filename from LIMS. Return an empty string if no target bed is found in LIMS.
         Raises:
             CgDataError: if the bed target capture version is not found in StatusDB.
         """
         case: Case = self.store.get_case_by_internal_id(internal_id=case_id)
-        sample: Sample = case.links[0].sample
+        sample: Sample = case.samples[0]
         if sample.from_sample:
             sample: Sample = self.store.get_sample_by_internal_id(internal_id=sample.from_sample)
         target_bed_shortname: str | None = self.lims.capture_kit(lims_id=sample.internal_id)
         if not target_bed_shortname:
-            return None
+            return EMPTY_STRING
         bed_version: BedVersion | None = self.store.get_bed_version_by_short_name(
             bed_version_short_name=target_bed_shortname
         )
         if not bed_version:
             raise CgDataError(f"Bed-version {target_bed_shortname} does not exist")
         return bed_version.filename
 
-    @staticmethod
-    def _get_germlinecnvcaller_flag(analysis_type: str) -> bool:
-        """Return True if the germlinecnvcaller should be skipped."""
-        return analysis_type == AnalysisType.WGS
+    def _create_sample_mapping_file(self, case_id: str, case_path: Path) -> Path:
+        """Create a sample mapping file for the case and returns its path."""
+        file_path = Path(case_path, f"{case_id}_customer_internal_mapping").with_suffix(
+            FileExtensions.CSV
+        )
+        content: list[list[str]] = [["customer_id", "internal_id"]]
```

Comment:
```suggestion
        headers: list[str] = ["customer_id", "internal_id"]
        content: list[list[str]] = [headers]
```
Just makes it a tad clearer for me but feel free to resolve if you feel it does not matter